SchoolManagementSystemApi/Model/TimeTable.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.ComponentModel.DataAnnotations;
```

Comment:
@New-Anointing As mentioned before, you don't need both a virtual property and a new property repeated